server/config.js

```diff
@@ -5,7 +5,8 @@ const cache = new Map()
 
 const defaultConfig = {
   webpack: null,
-  poweredByHeader: true
+  poweredByHeader: true,
+  assetPrefix: '/'
```

Comment:
I'm wondering if this should be a function that returns the prefix. Something like:

```js
assetPrefix ({ buildId }) {
  return `/_next/${buildId}/`
}
```